code/modules/mob/mob.dm

```diff
@@ -1472,3 +1472,56 @@
 		return
 	var/obj/screen/zone_sel/selector = mob.zone_sel
 	selector.set_selected_zone(next_in_list(mob.zone_sel.selecting,zones))
+
+//Height String for examine - Runs on the mob being examined.
+/mob/proc/assembleHeightString(mob/examiner)
+	var/heightString = null
+	var/descriptor
+	if(height == HEIGHT_NOT_USED)
+		return heightString
+
+	// Compare to Species Average
+	if(ishuman(src))
```

Comment:
As above, avoid `istype(src, ...) checks. This part can easily be a different proc that is overridden on different subtypes of /mob/.